lib/ansiblelint/utils.py

```diff
@@ -781,13 +799,10 @@ def get_yaml_files(options):
             universal_newlines=True
         ).split()
     except subprocess.CalledProcessError as exc:
-        if options.verbosity:
-            print(
-                "Warning: Failed to discover yaml files to lint using git:"
-                " {error_msg}".format(
-                    error_msg=exc.output.rstrip('\n')
-                )
-            )
+        logger.warning(
+            "Failed to discover yaml files to lint using git: %s",
+            exc.output.rstrip('\n')
+        )
```

Comment:
Note: #738 adds a new print statement below this place, be sure to replace it if it's merged first.

Reply:
@ssbarnea it's merged. And besides adding an extra except block, it also has tests that check stdout so you'll need to adjust them as well.